src/CalculatorUnitTests/CalculatorManagerTest.cpp

```diff
@@ -752,6 +752,11 @@ namespace CalculatorManagerTest
 
         Command commands10[] = { Command::ModeProgrammer, Command::Command1, Command::CommandRORC, Command::CommandRORC, Command::CommandNULL };
         TestDriver::Test(L"-9,223,372,036,854,775,808", L"RoR(RoR(1))", commands10, true, false);
+
```

Comment:
Suggest adding a few more test cases mentioned in #1788

Reply:
Sounds good, I'll add more test cases.

Reply:
I've added 3 test cases from #1788 and #1583